core/src/test/java/software/amazon/awssdk/core/pagination/PaginatedItemsIterableTest.java

```diff
@@ -68,7 +68,6 @@ public void hasNext_ReturnsFalse_WhenItemsAndPagesIteratorHasNoNextElement() {
     @Test
     public void hasNext_ReturnsTrue_WhenItemsIteratorHasNextElement() {
         when(singlePageItemsIterator.hasNext()).thenReturn(true);
-        when(pagesIterator.hasNext()).thenReturn(false);
```

Comment:
Why is this being removed?

Reply:
Mockito 2.x detects unused stubbings.  This line was flagged as an unused stubbing by the Mockito engine. see: https://static.javadoc.io/org.mockito/mockito-core/2.15.0/org/mockito/exceptions/misusing/UnnecessaryStubbingException.html

Reply:
This is what I encounter if I restore this 1 line and run it under Mockito 2.x:

Tests run: 7, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 11.459 sec <<< FAILURE! - in software.amazon.awssdk.core.pagination.PaginatedItemsIterableTest
unnecessary Mockito stubbings(software.amazon.awssdk.core.pagination.PaginatedItemsIterableTest)  Time elapsed: 0.332 sec  <<< ERROR!
org.mockito.exceptions.misusing.UnnecessaryStubbingException:

Unnecessary stubbings detected in test class: PaginatedItemsIterableTest
Clean & maintainable test code requires zero unnecessary code.
Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at software.amazon.awssdk.core.pagination.PaginatedItemsIterableTest.hasNext_ReturnsTrue_WhenItemsIteratorHasNextElement(PaginatedItemsIterableTest.java:71)
Please remove unnecessary stubbings or use 'lenient' strictness. More info: javadoc for UnnecessaryStubbingException class.